src/main/resources/db/migration/0300/V333__ObservationPlotBiomassDetails.sql

```diff
@@ -0,0 +1,212 @@
+CREATE TABLE tracking.biomass_forest_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+-- This is also in R__TypeCodes.sql, but is repeated here to ensure migration is successful
+INSERT INTO tracking.biomass_forest_types (id, name)
+VALUES (1, 'Terrestrial'),
+       (2, 'Mangrove')
+ON CONFLICT (id) DO UPDATE SET name = excluded.name;
+
+CREATE TABLE tracking.mangrove_tides(
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE
+);
+
+-- This is also in R__TypeCodes.sql, but is repeated here to ensure migration is successful
+INSERT INTO tracking.mangrove_tides (id, name)
+VALUES (1, 'High'),
+       (2, 'Low')
+ON CONFLICT (id) DO UPDATE SET name = excluded.name;
+
+CREATE TABLE tracking.observation_biomass_details (
+    observation_id BIGINT NOT NULL REFERENCES tracking.observations ON DELETE CASCADE,
+    monitoring_plot_id BIGINT NOT NULL REFERENCES tracking.monitoring_plots ON DELETE CASCADE,
+    description TEXT,
+    forest_type_id INTEGER NOT NULL REFERENCES tracking.biomass_forest_types,
+    small_trees_count_low INTEGER NOT NULL CHECK ( 0 <= small_trees_count_low),
```

Comment:
I think this check constraint would be slightly easier to read as `small_trees_count_low >= 0`.